component/loki/source/docker/runner.go

```diff
@@ -0,0 +1,236 @@
+package docker
```

Comment:
Just curious: did you find pkg/runner here useful or was there still too much boilerplate?